run-reframe

```diff
@@ -121,8 +121,10 @@ git clone https://github.com/reframe-hpc/reframe.git
 export PATH="$(pwd)/reframe/bin:$PATH"
 
 log "clone cscs-reframe-tests"
+log "WARNING: currently not using the alps branch - revert before merging"
 rm -rf cscs-reframe-tests
-git clone -b alps https://github.com/eth-cscs/cscs-reframe-tests.git
+git clone -b alps_santis https://github.com/jgphpc/cscs-reframe-tests.git
+#git clone -b alps https://github.com/eth-cscs/cscs-reframe-tests.git
 
 export UENV="${squashfs_path}:${mount}"
 export RFM_AUTODETECT_METHODS="cat /etc/xthostname,hostname"
```

Comment:
I think we need an additional variable here, namely [RFM_USE_LOGIN_SHELL](https://reframe-hpc.readthedocs.io/en/stable/manpage.html#envvar-RFM_USE_LOGIN_SHELL), otherwise the "command" `module` is unknown.
I think adding the line would work:
```bash
export RFM_USE_LOGIN_SHELL=1
```